building.html

```diff
@@ -151,11 +151,49 @@ <h3 id="building-windows">Building on Windows</h3>
       </ol>
 
 
-      <h3 id="building-nix">Building on macOS &amp; Linux</h3>
+      <h3 id="building-mac">Building on macOS</h3>
 
-      <p>Linux and macOS are <strong>not officially supported targets</strong>, and the game client does not compile. However, several of the tools can be built and run.</p>
+      <p>Compiling on macOS requires a minimum of macOS 11 and the Xcode build system. Xcode is available as a free download from the <a href="https://apps.apple.com/us/app/xcode/id497799835?mt=12/">App Store</a>.</p>
 
-      <p>You will need to ensure all the required dependencies are available. On macOS, a Homebrew bundle file is included in the repository, which can be installed by running <code>brew bundle</code>. You can also use vcpkg on these platforms for automatically managing dependencies.</p>
+      <p>We strongly encourage using vcpkg to automatically build dependencies; however, this requires some tools be manually installed in advance. The required tools are CMake, nasm, and pkg-config, and can be installed using common package managers such as Homebrew or MacPorts.</p>
+
+      <p>For Homebrew users who wish to manage all dependencies, a Homebrew bundle file is included in the repository, which can be installed by running <code>brew bundle</code> after cloning the project.</p>
+
+      <ol>
+        <li>
+          <p>Clone the repository from GitHub, including all submodules, using a git client. On the command line, this can be done by executing the following:<br>
+            <code>git clone --recurse-submodules https://github.com/H-uru/Plasma.git</code></p>
+        </li>
+
+        <li>
+          <p>In the folder where you cloned the repository, run CMake to create a build folder:<br>
+            <code>cmake -G Xcode -DUSE_VCPKG=ON -B build -S .</code></p>
+          <p>This assumes you wish to use vcpkg to build dependencies. Remove <tt>-DUSE_VCPKG=ON</tt> to manage dependencies yourself.</p>
+        </li>
+
+        <li>
+          <p>Compile the project with the following command:<br>
+            <code>cmake -B build --config RelWithDebInfo</code></p>
```

Comment:
Is `-B` the correct option here? The option for *running* the build is `--build`. `-B` is for setting the build directory when *configuring* the build.

```suggestion
            <code>cmake --build build --config RelWithDebInfo</code></p>
```